development/Obtain_dataset.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Load the dataset\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run this notebook, you will need to install: pandas, openai, transformers, plotly, matplotlib, scikit-learn, torch (transformer dep), torchvision, and scipy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\Lars\\AppData\\Local\\Temp\\ipykernel_13256\\3593873765.py:40: SettingWithCopyWarning: \n",
+      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
+      "Try using .loc[row_indexer,col_indexer] = value instead\n",
+      "\n",
+      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
+      "  data[\"Embeddings\"] = embeddings\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import openai\n",
+    "\n",
+    "# Setup API-Key for OpenAi\n",
+    "openai.api_key = \"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\"\n",
+    "\n",
+    "# Define the embedding model which is used\n",
+    "embedding_model = \"text-embedding-ada-002\"\n",
+    "\n",
+    "# Retrieve the data from the database\n",
+    "input_datapath = pd.read_sql_query(\"SELECT * FROM 'mortgage complaints'\", \"sqlite:///StaterData.db\")\n",
+    "\n",
+    "# Limit test size due to performance issues\n",
+    "data = input_datapath.loc[:100]\n",
```

Comment:
Why not only read 100 rows in your read_sql_query. Saves memory

Reply:
I think it is okay to not adjust the ``read_sql_query`` as we will combining our models soon. 
In the combined version we will only call the ``read_sql_query`` one time to prevent calling the same method multiple times. The ``input_datapath.loc[:100]`` can then stay.